index.html

```diff
@@ -256,26 +255,34 @@ <h3 class="ibm-padding-top-1 ibm-bold">UX Researchers</h3>
 
                                         <div class="ibm-col-12-4 ibm-col-medium-12-6">
                                             <div class="ibm-card ibm-card--noborder">
-                                                <div class="ibm-blocklink ibm-padding-content">
+                                                <a href="https://careers.ibm.com/ShowJob/Id/799683/Business-Analyst/?lang=en" id="Business-Analyst" target="_blank" rel="noopener noreferrer" class="ibm-blocklink ibm-padding-content">
                                                     <img src="assets/images/ba-blue.svg" alt="" class="ibm-padding-bottom-1">
-                                                        <h3 class="ibm-padding-top-1 ibm-padding-bottom-0 ibm-bold">Business Analyst</h3>
-                                                        <h4 class="ibm-padding-top-0">Roles now filled.</h4>
+                                                    <h3 class="ibm-padding-top-1 ibm-padding-bottom-0 ibm-bold">Business Analyst</h3>
                                                     <p class="ibm-textcolor-blue-80 ibm-light ibm-padding-top-1 ibm-padding-bottom-2">You’re an analytical thinker and negotiator who effectively communicates complex ideas to both technical and non-technical audiences. You maintain your team’s focus on delivering to business results, and are committed to becoming a trusted partner of the product owner.</p>
-
-                                                </div>
+                                                    
+                                                    <p classj="ibm-ind-link"><span class="ibm-forward-link ibm-btn-sec ibm-btn-blue-60">See full job description</span></p>
```

Comment:
You've got a `classj` here instead of `class`

Reply:
thank you.